lib/linalg/svd/IncrementalSVD.h

```diff
@@ -118,8 +115,7 @@ class IncrementalSVD : public SVD
     virtual
```

Comment:
Mentions of `time` should be deleted from the doxygen comment at line 110 and 113.